web/client/components/misc/GeneralButton.jsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+const {Button} = require('react-bootstrap');
+
+const GeneralButton = props => {
```

Comment:
We don't need this component use directly Button of react-bootstrap in the other component